code/modules/modular_computers/computers/machinery/modular_console.dm

```diff
@@ -37,7 +37,7 @@
 	var/obj/item/computer_hardware/network_card/wired/network_card = new()
 
 	cpu.install_component(network_card)
-	cpu.install_component(new /obj/item/computer_hardware/recharger/APC)
+	cpu.install_component(new /obj/item/computer_hardware/recharger/apc_recharger)
```

Comment:
You can run a quick regex, ie `(/datum).*[A-Z](/)` to catch a few more occurrences of these, in case the linter doesn't pan out. There's at least 50 occurrences or so from a quick check on my end